src/Symfony/Component/Console/Application.php

```diff
@@ -650,7 +647,9 @@ public function all($namespace = null)
             $commands = $this->commands;
             foreach ($this->commandLoader->getNames() as $name) {
                 if (!isset($commands[$name])) {
```

Comment:
what about

```php
if (!isset($commands[$name]) && $this->has($name)) {
    $commands[$name] = $this->get($name);
}
```

looks weird, but avoids repeating isEnabled logic here and there.

Reply:
👍  done